src/app/workflow/launch-third-party/filterCloudInstances.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { CloudInstance } from '../../shared/openapi';
+
+@Pipe({
+  name: 'filterCloudInstances',
+})
+export class FilterCloudInstancesPipe implements PipeTransform {
+  transform(cloudInstances: Array<CloudInstance>, filterFor: string): Array<CloudInstance> | null {
+    if (cloudInstances && filterFor) {
```

Comment:
Is it possible for cloudInstances to exist and filterFor to not be set? In that case, wouldn't you want to return `cloudInstances`? Right now it would return null.